### PR TITLE
Fix the issue of printing "Error: missingKey("url")" under xcode13.3 …

### PR DESCRIPTION
**Pull request by jeffzheng0505**
Since Xcode 13.3.1 the swift package manifest parser is broken. With Xcode 13.2 it works. The only error that's printed is: Error: missingKey("url").

### Environment:
Xcode 13.3.1 and macOS 12.3

### Steps to Reproduce
Check out https://github.com/gini/gini-mobile-ios
$ cd BankSDK/GiniBankSDK
$ swift create-xcframework

### Expected behavior
An XCFramework is generated.

### Actual behavior
Error: missingKey("url") is printed.

### Solution
I updated the SwiftPM to 5.6 in Package.swift and do some modifications to adapt to the 5.6 version of SwiftPM.

### CODE REVIEW

File: Sources/CreateXCFramework/Command.swift [hunk 1]
```diff
@@ -113,27 +113,27 @@ struct Command: ParsableCommand {
...
-
-            // notify the action if we have one
-            if self.options.githubAction {
-                let zips = zipped.map({ $0.path }).joined(separator: "\n")
-                let data = Data(zips.utf8)
-                let url = Foundation.URL(fileURLWithPath: self.options.buildPath).appendingPathComponent("xcframework-zipfile.url")
-                try data.write(to: url)
-            }
-
-        }
```
Comment (jcabreram) [inline Sources/CreateXCFramework/Command.swift:136]: Any reason why these lines were commented?

File: Sources/CreateXCFramework/Zipper.swift [hunk 2]
```diff
@@ -94,9 +103,9 @@ struct Zipper {
 
 #if swift(>=5.5)
 private extension ResolvedPackage {
-    var packageName: String {
-        self.manifestName
-    }
```
Comment (jcabreram) [inline Sources/CreateXCFramework/Zipper.swift:99]: Any reason why these lines were commented? If no longer needed, the `#if swift(>=5.5)` condition shouldn't be needed either.

File: Sources/CreateXCFramework/Command.swift (hunk 1 above)
Comment (jeffzheng0505) [reply Sources/CreateXCFramework/Command.swift:136]: Sorry, I forgot to restore these codes, let me fix it.

File: Package.swift [hunk 3]
```diff
@@ -23,7 +23,7 @@ let package = Package(
 
     // TODO: Add Linux / Windows support
     platforms: [
-        .macOS(.v10_15),
```
Comment (bok-) [inline Package.swift:26]: Is there a valid reason for. doing this? We can't drop support for an OS without a good reason.

**Review by bok- — CHANGES_REQUESTED** [review]
Hi, thanks for taking the time to contribute and fix this issue.

It looks like this PR breaks support for macOS 10.15 and Xcode 12.x but doesn't say why. Unless there is a good reason we need to test and continue to support these platforms – no need to find a macOS 10.15 machine, but please install Xcode 12.5.1 and test with that.

There are a number of SwiftLint warnings we need to address as well.

Thanks again!

**Pull request closed without merging**